framework/include/loops/ProjectMaterialProperties.h

```diff
@@ -39,7 +39,10 @@ class ProjectMaterialProperties : public ThreadedElementLoop<ConstElemPointerRan
 
   virtual void subdomainChanged() override;
   virtual void onElement(const Elem * elem) override;
-  virtual void onBoundary(const Elem * elem, unsigned int side, BoundaryID bnd_id) override;
+  virtual void onBoundary(const Elem * elem,
+                          unsigned int side,
+                          BoundaryID bnd_id,
+                          const Elem * /* lower_d_elem = nullptr*/) override;
```

Comment:
defaults aren't actually inherited from base classes. I would uncomment the entire thing here. Only comment out in the .C implementation if it's unused